<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Panel</class>
 <widget class="QWidget" name="Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 12pt &quot;等线&quot;;</string>
  </property>
  <widget class="QWidget" name="controller" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>190</width>
     <height>330</height>
    </rect>
   </property>
   <widget class="QPushButton" name="tempUp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>温度 +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tempDown">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>温度 -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="windUp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>风速 +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="windDown">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>风速 -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开关</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="viewer" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="illustrationWidget" native="true">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>9</y>
     <width>171</width>
     <height>331</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>当前温度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="cost">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- -</string>
    </property>
   </widget>
   <widget class="QLabel" name="power">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>风速：</string>
    </property>
   </widget>
   <widget class="QLabel" name="temperature">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>模式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="working">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>计费：</string>
    </property>
   </widget>
   <widget class="QLabel" name="expectedTemp">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- -</string>
    </property>
   </widget>
   <widget class="QLabel" name="mode">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- -</string>
    </property>
   </widget>
   <widget class="QLabel" name="windSpeed">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>送风状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>目标温度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="originalTemp">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>- -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>外温：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>功率：</string>
    </property>
   </widget>
  </widget>
  <zorder>controller</zorder>
  <zorder>illustrationWidget</zorder>
  <zorder>viewer</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
